en/docs/develop/components/api-proxies/policies.md

```diff
@@ -0,0 +1,88 @@
+# Policies
+
+Policies are units of business logic that a user can apply to slightly change the  `Request`, `Response`, or `Error` flow of an API invocation before reaching the backend or the client. For example, you can modify the response of an API invocation by adding a policy to the Response flow to transform the payload from JSON to  XML and to add a header in the response flow. 
+
+Choreo includes a default set of policies covering most of the common use cases. You can attach a policy or multiple policies easily through the Choreo Console. Furthermore, you can attach multiple policies, swap, and rearrange them conveniently.
+
+In Choreo, once you attach a mediation policy to a proxy, the deployment internally is a two-step process.
+
+1. Deployment initiation: generating the interceptor app 
```

Comment:
Well, generating the interceptor application is a part of the Deployment initiation, and committing and stuff would not look like a part of the generating an app. Hence used this.

Reply:
How about "Generating the interceptor application to initiate deployment"?

Another minor concern here is that although the leading text of the bullet point is similar to a title and therefore, it feels better to avoid using a punctuation mark there.